Update square-root description.md (#2495)
exercism/problem-specifications: exercises/square-root/introduction.md

# Introduction

We are launching a deep space exploration rocket and we need a way to make sure the navigation system stays on target.

> As the first step in our calculation, we take a number and find its square root—that is, the number that, when multiplied by itself, equals the given number.

Reviewer: ```suggestion
As the first step in our calculation, we take to add functionality that can take a target number and find its square root — (i.e. the number that, when multiplied by itself, equals the target number).
```

I got a bit lost in the definition of the square root with the use of "number" to mean different numbers. It's correct, but a little hard to parse. So I think labelling the "target number" (better name welcome) makes it clearer. I'd also rather put the definition in brackets as, while I'm very :+1: on having it, I think most people know what a square root is, and it's potentially a little confusing having to read it.

Reviewer 2: Is "take to add"  correct?  Seems to be a missing word in there.

The journey will be very long.

> To make the batteries last as long as possible, we had to make our rocket’s onboard computer very power efficient.
> Unfortunately that means that we can’t rely on fancy math libraries and functions, as they use more power.

Reviewer 3: ```suggestion
To make the batteries last as long as possible, we had to make our rocket's onboard computer very power efficient.
Unfortunately that means that we can't rely on fancy math libraries and functions, as they use more power.
```

> Instead we want to calculate the square root directly.

Reviewer 4: Note, "directly" suggests that the library functions do something less direct than the suggested algorithms. Using the library functions is just as direct implementing this yourself. 

We may want to consider replacing "directly" with something more accurate, like suggesting that we want to build a minimal math library that has only the one function we actually need: square root. 

Or we can leave it as is for now.

> Instead we want to calculate or approximate the square root directly.

Reviewer: Are we calculating or approximating? If we're approximating I think we need that to be clearer in the statement above.

# Introduction

We are launching a deep space exploration rocket and we need a way to make sure the navigation system stays on target.

> The first step in our coordinate calculation involves taking a number as input and finding another number which multiplied by itself will equal our input number, or in other words, finding the square root of our input number.

Reviewer 4: This sentence is hard for me to follow, even as a native English speaker. It might be worth trying to break this into smaller pieces.

> Because the journey will be very long, we had to make our rocket’s onboard computer very power efficient so it wouldn’t drain the batteries.

Reviewer 4: ```suggestion
The journey will be very long.
To avoid draining the battery, ee had to make our rocket’s onboard computer very power efficient.
```

> Unfortunately that means that we can’t rely on fancy math libraries and functions, as they would take more power.

Reviewer 4: ```suggestion
Unfortunately that means that we can’t rely on fancy math libraries and functions, as they use more power.
```

> The first step in our coordinate calculation involves taking a number as input and finding another number which multiplied by itself will equal our input number, or in other words, finding the square root of our input number.

Reviewer 2: ```suggestion
As the first step in our calculation, we take a number and find its square root—that is, the number that, when multiplied by itself, equals the given number.
```
